.devcontainer/devcontainer.json

```diff
@@ -26,7 +26,7 @@
   "waitFor": "postCreateCommand", // otherwise automated jest tests fail
   "features": {
     "node": {
-      "version": "14"
+      "version": "16"
```

Comment:
Can we go directly to Node 18?

Reply:
I'll try it this evening but I think it should work if Docusaurus works on NodeJS 18.

Reply:
Yes it works and I already pushed it!
![Screenshot_20220706-102832_Termius](https://user-images.githubusercontent.com/48137077/177506230-adebc2d7-520e-4c30-ab3b-b04803fc8a5e.jpg)